pkg/webreg/webreg.go

```diff
@@ -403,6 +404,231 @@ const templateDefinitions = `
+operator and <code>ci-operator</code> is able to build ephemeral versions of them
+usable for installation and testing, but not for production,
+</p>
+
+<h4 id="bundles"><a href="#bundles">Building Operator Bundles</a></h4>
+
+<p>
+Configuring <code>ci-operator</code> to build operator bundles from a repository
+is as simple as adding a new <code>operator</code> stanza, specifying the bundles
+that can be built from the repository, and what sorts of container image pull
+specification substitutions are necessary during bundle build time. Substitutions
+allow for the operator manifests to refer to images that were built from the
+repository during the test or imported from other sources. The following example
+builds an operator, a bundle and replaces the operator's pull specification in
+manifests with the version built during the test:
+</p>
+
+<code>ci-operator</code> configuration:
+{{ yamlSyntax (index . "optionalOperatorBundleConfig") }}
+
+<p>
+When configuring a bundle build, two options are available:
+</p>
+
+<ul>
+  <li><code>dockerfile_path</code>: path to the Dockerfile that builds the bundle image, defaulting to <code>bundle.Dockerfile</code></li>
+  <li><code>context_dir</code>: base directory for the bundle image build, defaulting to the root of the source tree</li>
+</ul>
+
+<p>The <code>.operator.bundles</code> stanza is a list, so building multiple bundle
+images is supported.</p>
+
+<h4 id="index"><a href="#index">Building an Index</a></h4>
+
+<p>
+When <code>ci-operator</code> builds at least one operator bundle from a
+repository, it will also automatically build an ephemeral index image to package
+those bundles. The test workloads should consume the bundles via this index
+image. The index image is named <code>ci-index</code> and can be exposed to test
+steps via the <a href="/help/ci-operator#literal-references">dependencies</a> feature.
+</p>
+
+<h4 id="ci-index-jobs"><a href="#ci-index-jobs">Validating Bundle and Index Builds</a></h4>
+
+<p>
+Similarly to how the job generator automatically creates a <code>pull-ci-$ORG-$REPO-$BRANCH-images</code>
+job to test image builds when <code>ci-operator</code> configuration has an
+<code>images</code> stanza, it will also make a separate job that attempts to
+build the configured bundle and index images. This job, named <code>pull-ci-$ORG-$REPO-$BRANCH-ci-index</code>,
+is created only when an <code>operator</code> stanza is present.
+</p>
+
+<h3 id="tests"><a href="#tests">Running Tests</a></h3>
+
+<p>
+Once <code>ci-operator</code> builds the operator bundle and index, they are
+available to be used as a <code>CatalogSource</code> by OLM for deploying and
+testing the operator. The index image is called <code>ci-index</code> and can
+be exposed to multi-stage test workloads via the <a href="/help/ci-operator#literal-references">
+<code>dependencies</code> feature</a>:</p>
+
+Step configuration example:
+{{ yamlSyntax (index . "optionalOperatorIndexConsumerStep") }}
```

Comment:
nit: we quote string literals in the rest of the yamls on the site right now so this could also do that to match